guides/doc-Monitoring_Guide/master.adoc

```diff
@@ -0,0 +1,25 @@
+= Monitoring Red Hat Satellite
+:imagesdir: topics/images
+:monitoring-red-hat-satellite:
+:numbered:
+:context: monitoring-guide
+
+include::common/attributes.adoc[]
+include::topics/con_satellite_metrics_overview.adoc[leveloffset=+1]
+include::topics/con_performance-co-pilot.adoc[leveloffset=+1]
+include::topics/con_performance-metrics-domain-agents.adoc[leveloffset=+2]
+include::topics/proc_installing-pcp-packages.adoc[leveloffset=+1]
+include::topics/proc_configuring-pcp-data-collection.adoc[leveloffset=+2]
+include::topics/proc_enabling-access-to-metrics-via-web-ui.adoc[leveloffset=+2]
+include::topics/proc_verifying-pcp-configuration.adoc[leveloffset=+2]
```

Comment:
remove this file - it adds nothing and will link to real docs from a proper introduction

Reply:
Okay.